### PR TITLE
Add heuristic message.

### PR DESCRIPTION
**Pull request by BillyONeal**
As requested in https://github.com/microsoft/vcpkg/issues/20190#issuecomment-1092125077

### CODE REVIEW

**Comment by BillyONeal** [comment]
image

File: src/vcpkg/install.cpp [hunk 1]
```diff
@@ -814,7 +819,12 @@ namespace vcpkg::Install
             }
             else
             {
-                auto msg = msg::format(msgCMakeTargetsUsage, msg::package_name = bpgh.spec.name()).extract_data();
+                // clang-format off
+                // to preserve output lines
+                auto msg = msg::format(msgCMakeTargetsUsage, msg::package_name = bpgh.spec.name()).appendnl()
+                    .append_indent().append(msgCMakeTargetsUsageHueristicMessage).appendnl()
+                    .extract_data();
+                // clang-format on
```
Comment (strega-nil) [inline src/vcpkg/install.cpp:827]: I don't think the `clang-format` is necessary, but I'm not that opinionated on it.

**Review by strega-nil — APPROVED** [review]

File: src/vcpkg/install.cpp (hunk 1 above)
Comment (BillyONeal) [reply src/vcpkg/install.cpp:827]: Without that it formats like this, losing the division between lines of output:

image

I don't feel strongly about it either though.

Comment (BillyONeal) [reply src/vcpkg/install.cpp]: I avoided it in a different way.

Comment (vicroms) [reply src/vcpkg/install.cpp]: Maybe we need an `appendnl(message)`?

Comment (BillyONeal) [reply src/vcpkg/install.cpp]: @vicroms The cross product of different appends starts to get pretty big then; I think this is fine

**Review by vicroms — APPROVED** [review]

**Pull request merged**